ansible_base/authentication/authenticator_plugins/oidc.py

```diff
@@ -207,7 +207,9 @@ class AuthenticatorPlugin(SocialAuthMixin, OpenIdConnectAuth, AbstractAuthentica
     configuration_encrypted_fields = ['SECRET']
 
     def extra_data(self, user, backend, response, *args, **kwargs):
-        for perm in ["is_superuser", "is_system_auditor"]:
+        from ansible_base.lib.utils.settings import get_setting
```

Comment:
Does this have to be a local import or can it move up?

Reply:
This is in plugins, which is why I'm hesitant to do it top-level.

Reply:
ok, well I'll give it a shot, and I assume if it still passes checks that's ok.